mage_ai/frontend/components/actions/SuggestionsList.tsx

```diff
@@ -0,0 +1,137 @@
+import api from '@api';
+import Accordion, { AccordionPanel } from '@oracle/components/Accordion';
+import Spacing from '@oracle/elements/Spacing';
+import { useMemo, useState, useEffect } from 'react';
+import SuggestionRow from './SuggestionRow';
+
+export type SuggestionsListProps = {
+  featureSet: any;
+  featureSetId: any;
```

Comment:
We will need to add a `featureSet` interface later, but `featureSetId` type should be defined, like as `number`.